shell/platform/darwin/ios/framework/Headers/FlutterEngineGroup.h

```diff
@@ -19,6 +19,7 @@ NS_ASSUME_NONNULL_BEGIN
  * @warning This class is a work-in-progress and may change.
  * @see https://github.com/flutter/flutter/issues/72009
  */
+FLUTTER_EXPORT
```

Comment:
Why do we add `__attribute__((visibility("default")))` to classes in our public headers?  I wouldn't have thought that would be necessary but I don't know the history of this, is `FLUTTER_EXPORT` defined to something else in some cases?

Reply:
Nope, that's what it's defined as.  When we compile the library we have the default visibility set to private, it's a linker flag.  The reason for it is mostly around c++ code.  By being more restrictive on what symbols are public different optimizations can happen in the loader, on top of having a little bit more control over how people use your API.

Reply:
> we have the default visibility set to private

Ah, gotcha.